pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -377,7 +415,7 @@ async def clone_build_data(self, local_path: Path):
         await exectools.cmd_assert_async(cmd)
 
     async def update_build_data(self, advisories: Dict[str, int], jira_issue_key: Optional[str]):
-        if not advisories and not jira_issue_key:
+        if not (advisories or jira_issue_key or self.release_date):
```

Comment:
Just to confirm, we are returning `False` only if all 3 values, `advisories`, `jira_issue_key` and `self.release_date `are None/Empty?